include/api/CInferenceModelDefinition.h

```diff
@@ -57,7 +56,7 @@ class API_EXPORT CWeightedMode : public CAggregateOutput {
     //! Construct with a weight vector of \p size with all entries equal to \p weight.
     CWeightedMode(std::size_t size, double weight);
     void addToDocument(rapidjson::Value& parentObject, TRapidJsonWriter& writer) const override;
-    const std::string& stringType() override;
+    const std::string& stringType() const final;
```

Comment:
This makes me wonder whether this class should actually be final, in which case we could just change this back to override.

Reply:
In general, I wonder whether more of the classes related to the inference model schema should be marked as final.